packages/compass-aggregations/src/components/pipeline-builder-input-documents.tsx

```diff
@@ -101,12 +102,14 @@ function PipelineBuilderInputDocuments({
             size="small"
           ></Icon>
         </IconButton>
-        <Body className={headerTextStyles}>
-          <b>
-            {count} Document{count === 1 ? '' : 's'}
-          </b>{' '}
-          in the collection
-        </Body>
+        {count !== undefined && (
+          <Body className={headerTextStyles}>
```

Comment:
Might tweak this. Not sure what to do in the unlikely event that the count becomes available after the docs.